packages/end-to-end/lint-staged.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  '*.{js}': ['eslint --quiet --fix']
+};
```

Comment:
Ah sorry I missed something which I only realized in testing: if you are going let each package handle its own linting you do need the `prettier --write` in your configuration as well (to format the files). You can compare to the api `package.json` lint-staged configuration and the webapp `lint-staged.config.js`.